libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFUserAccountManagerTests.m

```diff
@@ -211,8 +204,6 @@ - (void)testMultipleAccounts {
     // Create 10 users
     [self createAndVerifyUserAccounts:10];
     NSFileManager *fm = [[NSFileManager alloc] init];
-    NSError *error = nil;
-    XCTAssertTrue([self.uam saveAccounts:&error], @"Unable to save user accounts: %@", error);
 
     // Ensure all directories have been correctly created
```

Comment:
This is logic that should be reasonably reworked and tested as necessary in the context of the default account persistence class, no longer inline in this test.